index.js

```diff
@@ -76,13 +76,17 @@ var InfluxDB = function (options) {
 }
 
 InfluxDB.prototype._parseResults = function (response, callback) {
+  var self = this
   var results = []
   _.each(response, function (result) {
     var tmp = []
     if (result.series) {
       _.each(result.series, function (series) {
         var rows = _.map(series.values, function (values) {
-          return _.extend(_.zipObject(series.columns, values), series.tags)
+          var columnsDesanitized = _.map(series.columns, self._desanitizeString)
+          var valuesDesanitized = _.map(values, self._desanitizeString)
+          var tagsDesanitized = _.forOwn(series.tags, self._desanitizeObjectIter.bind(self))
```

Comment:
Where is this output from? The [JSON line protocol was deprecated](https://docs.influxdata.com/influxdb/v0.13/write_protocols/json/).